test/unit/imgix-provider.test.jsx

```diff
@@ -0,0 +1,109 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import ReactImgix from "../../src/index"
+import { ImgixProvider } from "../../es/HOCs/imgixProvider"
+
+const providerProps = {
+  domain: "sdk-test.imgix.net",
+  sizes: "100vw",
+}
+
+const imageProps = {
+  src: "https://assets.imgix.net/examples/pione.jpg",
+  sizes: "50vw",
+}
+
+describe('ImgixProvider', () => {
+
+  test('should not have context value defined if Provider has no props', () => {
+    const wrappedComponent = (
+      <ImgixProvider>
+        <ReactImgix {...imageProps}/>
+      </ImgixProvider>
+    )
+
+    const expectedProps = {
+      children: < ReactImgix src="https://assets.imgix.net/examples/pione.jpg" sizes="50vw" />,
```

Comment:
```suggestion
      children: <ReactImgix src="https://assets.imgix.net/examples/pione.jpg" sizes="50vw" />,
```

Seems like this PR could use some Prettier TLC

Reply:
Ya, idk what's happening with prettier in this repo for me 🤔. Apols for the messy formatting.